Sources/eppo/ApiEndpoints.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import os
+
+/// Handles API endpoint URL construction with subdomain support.
+public class ApiEndpoints {
+    private static let logger: Logger = {
+        let bundleID = Bundle.main.bundleIdentifier ?? "com.eppo.sdk"
```

Comment:
I'm not sure about this - could it be our customers' app gets used?

Reply:
fixed